### ISSUE TITLE
Getting error when running setup.sh using ubuntu on a windows machine

### ISSUE THREAD

**Issue by philipdalen** [GoogleCloudPlatform/fourkeys#67]
Hey @dinagraves,

Thanks for all the work with this project, we are in the process of implementing it in our organization. 

I'm getting an error when running setup.sh (following you're [youtube](https://www.youtube.com/watch?t=76&v=VLVqRVX6Cg4&feature=youtu.be ) video and was hoping you could point me in the right direction. My environment are as follows

### **Operating System**
Running on Ubuntu 20.04.1 LTS

### **gcloud**
I want to make use of an already existing project called "bcc-fourkeys" in our gcloud instance
I confirmed that billing is enabled and that I have owner access
![image](https://user-images.githubusercontent.com/12196246/105635680-78b80b00-5e64-11eb-83b7-19138d4c1602.png)

### **executing setup.sh**
When I attempt to run setup.sh I get the following error
![image](https://user-images.githubusercontent.com/12196246/105635817-55419000-5e65-11eb-9fed-a9d686f10406.png)

Please let me know if you need an more information.
Greatly appreciate the help.

Kind Regards,
Philip

**Comment by davidstanke** [comment]
Hi Philip! Thanks for raising this issue and posting the screenshots. It looks like this is not a permissions issue, but rather a parsing problem -- it appears that the script is trying to operate against a GCP project named "bcc-fourkeys%0D", which is an invalid project name. 

Where did it get the idea to use that wonky name? From [here](https://github.com/GoogleCloudPlatform/fourkeys/blob/2e2e281f9ecec09f8d9743e6f16b8226aabf4261/setup/setup.sh#L23), which uses `printf` to format the name to use for projects to be created.

Why would it do that? Because [bash != sh](https://stackoverflow.com/questions/25894256/difference-in-calling-printf-from-sh-and-bash). :-/ The "printf" method has different behavior depending on the shell.

**TL;DR:** It looks like you're invoking setup by running `sh setup.sh`. **Try executing it as `./setup.sh` instead**, which will execute in bash, as per the script's shebang. Let us know if that works.

**Comment by philipdalen** [comment]
Thanks you very much for the prompt response @davidstanke. I found the problem now. Basically what happens is that I'm running ubuntu on a windows machine, I used git bash in windows to clone the repo and after that I using the ubuntu shell to run the setup.sh. The problem is that when using windows to pull down the repo it uses a different format to indicate line endings, so when I execute the script in Linux it thinks the line endings is a syntax error. After I used Linux also to pull the down the repo everything worked.

The take away is that when using Linux on windows, do everything through Linux.

Thanks again for the help.
